### PR TITLE
More lax on supported expires types

### PR DESCRIPTION
**Pull request by mems**
Fix the error when the expires is a number as string

### CODE REVIEW

**Comment by carhartl** [comment]
We are already coercing the expires attribute like so: `attributes.expires * 864e+5` so I guess  changing the if condition from `if (typeof attributes.expires === 'number')` to `if (!(attributes.expires instanceof Date))` would be so sufficient. 

I’m just not sure relaxing the input type is a desirable thing, after all it’s documented and therefore expected (expectable?).

File: src/js.cookie.js [hunk 1]
```diff
@@ -52,8 +52,12 @@
...
 			}, api.defaults, attributes);
 
-			if (typeof attributes.expires === 'number') {
-				attributes.expires = new Date(new Date() * 1 + attributes.expires * 864e+5);
+			// Check Date validity
+			if (attributes.expires instanceof Date) {
+				attributes.expires = isNaN(Number(attributes.expires)) ? '' : attributes.expires;
+			} else if (attributes.expires) {
+				var expires = parseFloat(attributes.expires);
+				attributes.expires = isNaN(expires) ? '' : new Date(new Date() * 1 + expires * 864e+5);
```
Comment (FagnerMartinsBrack) [inline src/js.cookie.js]: Need a failing test to justify the code change, see https://github.com/js-cookie/js-cookie/blob/d2724a87a29cbf7ee539452fb42b3e3a1aff95a8/CONTRIBUTING.md#pull-requests

Comment (carhartl) [reply src/js.cookie.js]: I didn’t even mention lack of tests because I’m not sure we want to relax the api in the first place.

Comment (FagnerMartinsBrack) [reply src/js.cookie.js]: That's right, agree on that

**Comment by FagnerMartinsBrack** [comment]
Closing due to the lack of feedback from the [OP](https://medium.com/@fagnerbrack/what-does-op-means-on-github-8db958b604af). If you want to continue the discussion, please comment here mentioning one of the project members.

**Pull request closed without merging**